front/lib/api/user.ts

```diff
@@ -121,44 +101,24 @@ export async function setUserMetadata(
   await metadata.save();
 }
 
-export async function updateUserFullName({
-  user,
-  firstName,
-  lastName,
-}: {
-  user: UserType;
-  firstName: string;
-  lastName: string;
-}): Promise<boolean | null> {
-  const u = await User.findOne({
-    where: {
-      id: user.id,
-    },
+export async function fetchRevokedWorkspace(
+  user: UserTypeWithWorkspaces
+): Promise<Workspace | null> {
+  // TODO(@fontanierh): this doesn't look very solid as it will start to behave
+  // weirdly if a user has multiple revoked memberships.
+  const userRes = await UserResource.fetchNonNullableByModelId(user.id);
```

Comment:
`nit`
```suggestion
  const user = await UserResource.fetchNonNullableByModelId(user.id);
```

As the `Res` suffix is often use for our `Result`